src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs

```diff
@@ -504,13 +500,10 @@ public void Configure(ResourceManagementOptions options)
                     break;
             }
 
-            options.UseCdn = settings.UseCdn;
-
-            options.CdnBaseUrl = settings.CdnBaseUrl;
-
-            options.AppendVersion = settings.AppendVersion;
-
-            options.ContentBasePath = _httpContextAccessor.HttpContext.Request.PathBase.Value;
+            options.UseCdn = _resourceSetting.UseCdn;
+            options.CdnBaseUrl = _resourceSetting.CdnBaseUrl;
+            options.AppendVersion = _resourceSetting.AppendVersion;
+            options.ContentBasePath = _pathBase;
```

Comment:
_pathBase.Value

There is a lack of consistency here if we look in other places where we pass _pathBase.Value

Reply:
@Skrypt done.